code/game/gamemodes/miniantags/abduction/abduction.dm

```diff
@@ -223,6 +223,7 @@
 
 	var/obj/item/device/radio/R = new /obj/item/device/radio/headset/syndicate/alt(agent)
 	R.set_frequency(radio_freq)
+	R.name = "strange headset"
```

Comment:
Everything else related to abductors is called "alien".

"alien drill", "alien scalpel", "alien pistol", "alien table", "alien locker".